src/sidecar/authStatusPolling.test.ts

```diff
@@ -172,61 +169,3 @@ describe("CCloud auth expiration checks", () => {
     assertAuthExpiredPromptNotOpened();
   });
 });
-
-describe("CCloud connection status polling", () => {
```

Comment:
Removed entirely since the poller doesn't control invalidating the current CCloud auth session; handled by middleware now